draft-wang-ppm-dap-taskprov.md

```diff
@@ -579,13 +579,27 @@ Author to be under control of the adversary. It is therefore incumbent on
 protocol participants to verify the privacy parameters of a task before opting
 in.
 
-Another risk is that a malicious coalition of Clients might attempt to pollute
-an Aggregator's long-term storage by uploading reports for many (thousands or
-perhaps millions) of distinct tasks. While this does not directly impact tasks
-used by honest Clients, it does present a Denial-of-Service risk for the
-Aggregators themselves. This can be mitigated by limiting the rate at which new
-tasks or configured. In addition, deployments SHOULD arrange for the Author to
-digitally sign the task configuration so that Clients cannot forge task creation.
+Another risk is that the Author could configure a unique task to fingerprint a
+Client. Although Client anonymization is not guaranteed by DAP, some systems
+built on top of DAP may hope to achieve this property by using a proxy server
+with Oblivious HTTP {{!I-D.draft-ietf-ohai-ohttp-08}} to forward Client reports
```

Comment:
:)

```suggestion
with Oblivious HTTP {{!RFC9458}} to forward Client reports
```

Reply:
I copied it from DAP draft, perhaps this needs to be updated in DAP as well.